youtube_dl/extractor/bandlab.py

```diff
@@ -0,0 +1,172 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+from ..compat import (
+        compat_str,
+)
+
+from ..utils import (
+    strip_or_none,
+    try_get,
```

Comment:
Let's use `traverse_obj()` now.
```suggestion
    T,
    traverse_obj,
```